### PR TITLE
#4274 broken mobile fixes

### LINKED ISSUE #4274: Broken centering on landing page on mobile at less than 1023px
Seems it's broken centering between 669px and 1023px

image

and mobile over 1024px is broken can't see whole navbar

image

overflowing on mobile 

image

plus when you scale with fingers on mobile it's looks like this and I guess should not. 

telegram-cloud-photo-size-4-5998822561271626104-y

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## PR Context?

- [x] Closes #4274
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by prachi00** [comment]
Screen Shot 2022-11-06 at 3 19 36 PM
@yangwao couldn't reproduce this one

File: components/landing/LandingPage.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div>
+  <div class="overflow-hidden">
```
[the remark this answers is not in the document]
Comment (prachi00) [reply components/landing/LandingPage.vue]: sure, removed this, seems to work fine

File: styles/global.scss [hunk 2]
```diff
@@ -123,6 +123,10 @@ body {
   }
 }
 
+// .overflow-hidden {
```
[the remark this answers is not in the document]
Comment (prachi00) [reply styles/global.scss]: oh yes my bad, removed

**Review by kkukelka — APPROVED** [review]

**Comment by kkukelka** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 29.36 USD/KSM ~ 1.022 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x7f25c87c5128e57133c250475c1f8ea1afbfa99f389a9ebd9d74677c88bebaee](https://kusama.subscan.io/extrinsic/0x7f25c87c5128e57133c250475c1f8ea1afbfa99f389a9ebd9d74677c88bebaee)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**